include/abb_librws/rws_interface.h

```diff
@@ -512,6 +512,16 @@ class RWSInterface
    */
   bool endSubscription();
 
+  /**
+   * \brief Close the active subscription connection.
```

Comment:
```suggestion
   * \brief Force close an active subscription connection.
```
Slightly more accurate description.